int32 yaw_start
int32 yaw_stop
int32 pitch_start
int32 pitch_stop